### ISSUE TITLE
Collaborate Hudi Pyspark blog

### ISSUE THREAD

**Issue by bhasudha** [vasveena/Hudi_Demo_Notebook#2]
Hi there,

I came across this blog - https://github.com/vasveena/Hudi_Demo_Notebook/blob/master/apache-hudi-on-amazon-emr-pyspark-demo-shareable.ipynb . It looks really cool and well done. IMO, this will be very useful for a lot of pyspark Hudi users. I was wondering if its okay to link to this blog from Hudi website. Since I couldnt find a way to contact, I am reaching out here. 
Please let me know.

**Comment by vasveena** [comment]
Hi Bhavani,

My sincere apologies for the delay in response. I’m glad to know that the notebook is useful. I would be more than happy for you to link it to from Hudi website.

I’m working on checking in this code to our solution architects platform. Would you mind waiting until early next week so that I can get you the updated version of this notebook?

Best Regards,
Veena

**Comment by bhasudha** [comment]
Absolutely. Thanks much @vasveena. This is great work. I ll wait to hear from you!
